.github/workflows/on_push.yml

```
@@ -22,3 +22,8 @@ jobs:
         run: |
           pytest . -vv --ignore-glob="venv/**/*" --durations=0 --cov --cov-branch --cov-report=xml
           bash <(curl -s https://codecov.io/bash)
+      - name: Deploy to Google App Engine
+        uses: atRobertoFlores/gae_deploy_action@master
```

googleが提供している方がよくない？
```suggestion
        uses: GoogleCloudPlatform/github-actions/appengine-deploy@master
```
with以下のキー名も変わるけど、使い方は同じだと思う

cf. [github-actions/appengine-deploy at master · GoogleCloudPlatform/github-actions](https://github.com/GoogleCloudPlatform/github-actions/tree/master/appengine-deploy)

---

やってみます！ありがとうございます！